docs/components/contents.js

```diff
@@ -0,0 +1,11 @@
+export const contents = {
+  'Getting Started': ['Introduction', 'Overview'],
+  API: [
+    'Summary',
+    {
+      label: 'HelloWorldClass',
+      href: './generated/hello_project.HelloWorldClass',
+    },
+    { label: 'hello_world', href: './generated/hello_project.hello_world' },
+  ],
```

Comment:
It should be possible to automatically populate the api members here yeah? I see a lot of value in being able to customize the sidebar but perhaps the default behavior could be the same list members that appear in the `api.fjson` file